app/front/scripts/services/visualizations/index.js

```diff
@@ -65,6 +65,13 @@ var availableVisualizations = [
     type: 'pivot-table',
     embed: 'pivottable',
     icon: 'os-icon os-icon-layers'
+  },
+  {
+    id: 'Radar',
+    name: 'Radar Chart',
+    type: 'pivot-table',
```

Comment:
Why is the type 'pivot-table'?
My first instinct is to make it a 'sortable-series', wdyt?